cos-fleetshard-operator-debezium-it/src/test/java/org/bf2/cos/fleetshard/operator/it/debezium/glues/KafkaConnectorSteps.java

```diff
@@ -176,6 +176,33 @@ public void kctr_status_is(String phase) {
                 return resource;
             });
     }
+    @When("the kctr status is RUNNING with a failed task")
+    public void kctr_with_failed_task() {
```

Comment:
Not an issue but wouldn't be better to make this step configurable ? so you can write something like:

```gherkin
 When the kctr is "RUNNING" with tasks status:
   | id | state  | trace                       |
   | 0  | FAILED | The connector is trying ... |
```

Reply:
Not really. The message is superfluous and doesn't have to be customized. I might revisit this when we look into multi-task connectors.

Reply:
Ok, since there were a very specific error message I thought it was meaningful and that such error was the one to catch. Maybe add a comment to say that error is not relevant